modules/ppcp-button/resources/js/modules/Renderer/CreditCardRenderer.js

```diff
@@ -122,16 +122,18 @@ class CreditCardRenderer {
 
             });
 
-            if (!this.formSubmissionSubscribed) {
-                document.querySelector(wrapper + ' button').addEventListener(
-                    'click',
-                    event => {
-                        event.preventDefault();
-                        this._submit(contextConfig);
-                    }
-                );
-                this.formSubmissionSubscribed = true;
-            }
+           const checkoutForm = document.forms.checkout;
```

Comment:
hm, does it work with mollie now? I thought we would need to add the attribute to something that gets reloaded during this switch (that is the card form).

Reply:
yup, it works adding the attribute to checkout form. 

I tried adding it to the hosted fields form wrapper (https://github.com/woocommerce/woocommerce-paypal-payments/blob/trunk/modules/ppcp-button/resources/js/modules/Renderer/CreditCardRenderer.js#L75) but it does not work.

It works on the hosted fields button wrapper (https://github.com/woocommerce/woocommerce-paypal-payments/blob/trunk/modules/ppcp-button/resources/js/modules/Renderer/CreditCardRenderer.js#L15) so this is what I used finally.